crates/wasmtime/src/module.rs

```diff
@@ -517,12 +519,14 @@ impl Module {
         mmap: MmapVec,
         info: Option<CompiledModuleInfo>,
         types: impl Into<Types>,
+        enable_branch_protection: bool,
```

Comment:
Could this be dropped here since it can be inferred from the `&Engine` argument?

Reply:
See my comment about line 540 in `crates/wasmtime/src/engine.rs`.